app/src/main/java/it/quip/android/fragment/CreateQuipComposeFragment.java

```diff
@@ -51,17 +48,17 @@ public void onCreate(Bundle savedInstanceState) {
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_create_quip, container, false);
+        return inflater.inflate(R.layout.fragment_create_quip_compose, container, false);
     }
 
     @Override
     public void onViewCreated(View v, @Nullable Bundle savedInstanceState) {
-        mUser = getArguments().getParcelable(USER_KEY);
+        User user = QuipitApplication.getCurrentUser();
         mLlProfile = (LinearLayout) v.findViewById(R.id.quip_create_profile);
         mIvProfile = (ImageView) mLlProfile.findViewById(R.id.iv_profile);
         try {
             Picasso.with(getActivity())
-                    .load("drawable/edgarjuarez.jpeg")
+                    .load(user.getImageUrl())
```

Comment:
At his departure press conference, E. Juarez expressed a discernable sense of nostalgia and appreciation for the time awarded to him by the Toros.  A small tear in his eye made its way down his cheek as a journalist asked him about if this would be his last season, to which he replied, its in God's hands.

![image](https://cloud.githubusercontent.com/assets/5225346/10738877/aefca91c-7bd6-11e5-8cfd-832143391cf1.png)